src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -151,12 +155,26 @@ public void addCountryNote(Country country, Note countryNote) {
 
         countryManager.addCountryNote(country, countryNote);
     }
+
     @Override
     public void addClientNote(Client target, Note clientNote) {
         requireAllNonNull(target, clientNote);
         target.addClientNote(clientNote);
     }
 
+    /**
+     * Initialises TagNoteMap from Clients notes and Country notes.
+     */
+    public void initialiseTagNoteMap() {
+        this.tagNoteMap.initMapFromClients(this.addressBook.getClientList());
+        this.countryManager.getAllCountryNotes().forEach(this.tagNoteMap::initMapFromCountries);
+    }
```

Comment:
This doesn't seem to be called in the program, will it be added in a later commit?

Reply:
was sitting on it because this depended on Rayson's Country class. Will add this part in since he's done.